index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Labrada:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title id="title">Ariel Camus CEO at Microverse</title>
+</head>
+
+<body>
+    <main id="main">
+        <!-- H E A D E R -->
```

Comment:
It would be more appropriate to use the header tag instead of a div

Reply:
Great job👏  on making a polite request for a change! it would be helpful to provide some context on why the change you're requesting is necessary. This can help the student understand the reasoning behind the change and prevent them from making similar mistakes in the future. By explaining the importance of the change, the student can also learn more effectively and improve their coding skills. Keep up the good work!👍